invenio_rdm_records/resources/serializers/ui/schema.py

```diff
@@ -106,7 +106,7 @@ def mask_removed_by(obj):
         if user == "system":
             return_value = _("System (automatic)")
         elif user is not None:
-            return_value = _("User")
+            return_value = _("Admin")
```

Comment:
could you verify if this is also applied when we delete a community?

Reply:
no this is not, will have to modify this in invenio-communities